### PR TITLE
Update ABOS talend jobs for new SOTS structure

### PR DESCRIPTION
**Pull request by mhidas**
This has already been done in prod (https://github.com/aodn/chef-private/pull/2496), forgot to do it for the po box.

### CODE REVIEW

File: private-sample/data_bags/talend/abos_sofs_fl.json [hunk 1]
```diff
@@ -14,7 +14,8 @@
     "event": {
         "regex": [
             "^IMOS/ABOS/ASFS/SOFS/Surface_fluxes/Real-time/.*\\.nc$",
-            "^IMOS/ABOS/ASFS/SOFS/Surface_fluxes/IMOS_ABOS.*\\.nc$"
+            "^IMOS/ABOS/ASFS/SOFS/Surface_fluxes/IMOS_ABOS.*\\.nc$",
+            "^IMOS/ABOS/SOTS/.*/IMOS_ABOS-[^_]*_[^_]*F.*(SOFS|FluxPulse)_FV02.*\\.nc$"
```
Comment (ggalibert) [inline private-sample/data_bags/talend/abos_sofs_fl.json]: Not exactly the same changes as in the other PR for prod where I find:
```json
"^IMOS/ABOS/SOTS/[0-9]{4}/real-time/IMOS_ABOS-ASFS_FMT_.*SOFS.*\\.nc$"
```

**Comment by ggalibert** [comment]
Apart for the comment above, happy to merge.

File: private-sample/data_bags/talend/abos_sofs_fl.json (hunk 1 above)
Comment (mhidas) [reply private-sample/data_bags/talend/abos_sofs_fl.json]: See https://github.com/aodn/chef-private/pull/2303, which has just been merged on top of https://github.com/aodn/chef-private/pull/2496. The combined effect of the two is what's applied here.

**Comment by mhidas** [comment]
@ggalibert  Thanks for checking. See comment above. It's good to go.

**Pull request merged**